apps/src/studio/skins.js

```diff
@@ -931,8 +931,47 @@ function loadHoc2015(skin, assetUrl) {
     'applause'
   ];
 
+  skin.soundGroups = {
+    'R2-D2sound': {
+      randomValue: 'R2-D2random',
+      minSuffix: 1,
+      maxSuffix: 9
```

Comment:
I'm a little confused by what `minSuffix` and `maxSuffix` mean here.  Is this because we have 9 R2-D2 sound effects, or do we expect no more than 9 additional characters in an R2-D2 sound effect name?  Either way, it seems like a thing we could derive, rather than define here.  What am I missing?